doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -145,6 +145,14 @@ Security
   * The :ref:`ug_crypto_supported_features` page with updated values for the supported cryptographic operations and algorithms.
     Now, the page only lists features and algorithms that are supported by at least one driver.
 
+* Updated the Oberon PSA Crypto to version 1.5.1 that introduces support for the following new features with the Oberon PSA driver:
+
+  * Experimental support for post-quantum cryptography schemes ML-KEM (FIPS 203), ML-DSA (FIPS 204) and NIST SP 800-208.
+  * Experimental support for XChaCha20-Poly1305 AEAD algorithm.
+  * Experimental support for SHAKE128 with 256-bit output length.
```

Comment:
```suggestion
    Now, the page only lists features and algorithms that are supported by at least one driver.
  * Updated the Oberon PSA Crypto to version 1.5.1 that introduces support for the following new features with the Oberon PSA driver:

    * Experimental support for post-quantum cryptography schemes ML-KEM (FIPS 203), ML-DSA (FIPS 204) and NIST SP 800-208.
    * Experimental support for XChaCha20-Poly1305 AEAD algorithm.
    * Experimental support for SHAKE128 with 256-bit output length.
```
Whoops, I noticed there is already the "heading" Updated above. Therefore, took out the word "Updated" and fixed the indentation.